### ISSUE TITLE
Visual Display for Branch Coverage

### ISSUE THREAD

**Issue by jflowers** [jmhofer/eCobertura#10]
This is a feature request.

I am not sure how to display this...

Right now only line coverage is shown through background colorization of lines covered when viewing source code.  We need a way to visualize the branch coverage when viewing the source code.

**Comment by jmhofer** [comment]
I agree. Imho, branch coverage is Cobertura's strength and it really should be visualized... somehow... this may take some time, but I'll see what I can come up with.

**Comment by vuillee** [comment]
I would be happy if a partially covered line just had a different background (e.g yellow or orange)

**Comment by nkerr** [comment]
Has this issue been resolved? It would be ideal if branch coverage would be rendered the same way that it is rendered in the sonar web GUI (http://jira.codehaus.org/secure/attachment/43696/screenshot-1.jpg), i.e. as a percentage in a column next to the line number on the left.

**Comment by kennethxu** [comment]
+1
it is a real pain now to find which branch I have missed.
